app/lib/benchmarks_fixture.rb

```diff
@@ -0,0 +1,35 @@
+class InvalidParameter < Exception; end
+class OutOfBounds < Exception; end
+
+class BenchmarksFixture
+  def initialize
+    @fixture =
+      JSON.load File.open './app/fixtures/benchmarks_and_activities.json'
+  end
+
+  def activities(benchmark_id, args)
+    raise "Unknown benchmark: #{benchmark_id}" unless @fixture[benchmark_id]
```

Comment:
This function is kind of annoying because the actual logic of what the function is doing has gotten lost in all of the parameter validation.

Reply:
I liked this and made .goal_activities and .capacity_activities on BenchmarksFixture.